quantus/metrics/axiomatic/completeness.py

```diff
@@ -255,8 +255,6 @@ def evaluate_instance(
         y: np.ndarray,
         a: np.ndarray,
         s: np.ndarray,
-        c: Any,
```

Comment:
Remove from docstring:

c: any
    The custom input to be evaluated on an instance-basis.
p: any
    The custom preprocess input to be evaluated on an instance-basis.

Reply:
Replace in all places in docstring so that we don't miss

Reply:
done